packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -10,6 +10,9 @@ import { IndexQueryNotAvailableError } from '../index-query-not-available.error.
 export class PostgresIndexApi implements DatabaseIndexApi {
   readonly insertionOrder: InsertionOrder
   readonly modelsToIndex: Array<StreamID> = []
+  // Maps Model streamIDs to the list of fields in the content of MIDs in that model that should be
+  // indexed
+  readonly modelsIndexedFields = new Map<string, Array<string>>()
```

Comment:
Grows linearly with number of streams we index. Guess, we'll address this in future.

Reply:
yeah.  I figure it'll be unlikely for apps to index more than a hundred models and in the very worst case maybe they'll have like a thousand, but even still those are pretty small numbers so I'm not too worried about this structure getting very big.